Framework/Kernel/test/MaterialBuilderTest.h

```diff
@@ -131,7 +131,7 @@ class MaterialBuilderTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(mat.numberDensity(), 0.0410414, 0.001);
   }
 
-  void test_Number_Set_By_AtomicNumber_ZParameter_And_Cell_Volume() {
+  void test_Number_Density_Set_By_AtomicNumber_ZParameter_And_Cell_Volume() {
```

Comment:
Since the rest of the file is in this format for function names, I am not too bothered. But the mantid standards state that:
![image](https://user-images.githubusercontent.com/24723609/53722021-77d6f400-3e5c-11e9-8b5d-14d3a5258e87.png)
So according to that it should be `test_numberDensitySetByAtomicNumberZParameterAndCellVolume`.

Again the whole file is at fault and that is outside the scope so leave the test names as they are :+1:

Reply:
Our documentation for unit test practices has still another style without the underscore:
http://developer.mantidproject.org/UnitTestGoodPractice.html

Traditionally one mixes all possible naming styles within the unit test files.

Reply:
I was following the example given in our standards:
http://developer.mantidproject.org/Standards/UnitTestStandards.html

e.g.:
![image](https://user-images.githubusercontent.com/24723609/53731852-f9864c00-3e73-11e9-9674-e39f5fd567b1.png)